app/src/main/java/org/mozilla/fenix/home/collections/Collection.kt

```diff
@@ -79,7 +80,13 @@ fun Collection(
     Card(
         modifier = Modifier
             .semantics(mergeDescendants = true) {}
-            .clickable { onToggleCollectionExpanded(collection, !isExpanded) }
+            .clickable(
+                onClickLabel = if (isExpanded) {
+                    stringResource(R.string.a11y_action_label_collapse)
+                } else {
+                    stringResource(R.string.a11y_action_label_expand)
+                },
+            ) { onToggleCollectionExpanded(collection, !isExpanded) }
```

Comment:
nit: we should try not to mix named arguments with "positional" ones. i.e.: also add named argument for `onClick`